cli/dcoscli/marathon/main.py

```diff
@@ -674,15 +674,27 @@ def _clean_up_resource_definition(properties):
     :rtype: dict
     """
     clean_properties = {}
+    ignore_properties = ["deployments", "lastTaskFailure", "tasks",
+                         "tasksRunning", "tasksHealthy", "tasksUnhealthy",
+                         "tasksStaged"]
```

Comment:
regarding the docstring above, this function no longer removes just task properties